src/appMain/sdl_preloaded_pt.json

```diff
@@ -538,6 +538,12 @@
                         "FULL",
                         "LIMITED",
                         "NONE"]
+                    },
+                    "OnRCStatus": {
+                        "hmi_levels": ["BACKGROUND",
+                        "FULL",
+                        "LIMITED",
+                        "NONE"]

```

Comment:
suggest to enable RC by default in sdl_preloaded_pt.json file by adding the following
                 "groups": ["Base-4", "RemoteControl"],
                "moduleType": [
                    "RADIO",
                    "CLIMATE",
                    "AUDIO",
                    "LIGHT",
                    "HMI_SETTINGS",
                    "SEAT"   ]
to section  "app_policies": {             "default": {